### PR TITLE
Convert to md web/api/event [es]

### PR DESCRIPTION
**Pull request by JuanVqz**
### Description

Converted from HTML to md

- [x] index.md
- [x] bubbles
- [x] cancelable
- [x] currenttarget
- [x] defaultprevented
- [x] event
- [x] initevent
- [x] preventdefault
- [x] stoppropagation
- [x] target
- [x] type

### Motivation

Help translating Spanish documentation

### Related issues and pull requests

Related to https://github.com/mdn/translated-content/issues/8635

### CODE REVIEW

File: files/es/web/api/event/event/index.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+## Especificaciones
+
+{{Specifications}}
+
+## Compatibilidad con navegadores
+
+{{Compat}}
+
+## Ver también
```
Comment (Graywolf9) [inline files/es/web/api/event/event/index.md:57]:
```suggestion
## Véase también
```

**Review by Graywolf9 — APPROVED** [review]
Excelente conversión, muchas gracias!

File: files/es/web/api/event/event/index.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
+---
+title: Event()
+slug: Web/API/Event/Event
+translation_of: Web/API/Event/Event
+browser-compat: api.Event.Event
+---
```
Comment (Graywolf9) [inline files/es/web/api/event/event/index.md:6]: Si ya esta actualizado al día, pondrás la etiqueta l10n-source?

Comment (JuanVqz) [reply files/es/web/api/event/event/index.md:6]: va que va!

**Pull request merged**